<launch> 
    <arg name="mission_files" default="default.json"/>
    <arg name="kml_files" default="default.kml"/> 
    <arg name="minimum_turning_radius" default="5"/>
    <arg name="path_resolution" default="0.1"/>
    

    <!-- <param name = "/gps_path_utils/mission_files" value = "$(arg mission_files)"/>  -->
    <param name = "/gps_path_utils/kml_file" value = "$(arg kml_files)"/> 
    <param name="minimum_turning_radius" value="$(arg minimum_turning_radius)"/>
    <param name="path_resolution" value="$(arg path_resolution)"/>
    <node pkg="gps_path_utils" type="kml_to_mission_file_circle_method.py" name="kml_to_mission_file_circle_method_node" output="screen"/>
</launch> 
